app/views/news/2018-09-06-august-2018-monthly-update.html.md

```diff
@@ -0,0 +1,72 @@
+---
+summary: In August, we paid for 113 hours of developer work, shipped a new Bundler guide, and fixed bugs in Bundler and RubyGems.
+---
+
+Hello! Welcome to the monthly update. During August, our work was supported by [Handshake](https://handshake.org), [reinteractive](https://reinteractive.com/), and many others.
+
+## ruby together news
+
+In August, Ruby Together was supported by 74 different companies. [Handshake](https://www.handshake.org) joined as a [LEVEL] member. In total, we were supported by 77 individual members and 64 friends of Ruby Together. Thanks to all of our members for making everything that we do possible.
+
+In community events, our developer evangelist PJ Hagerty spent some time at the Google Developer Group meetup in New York City, Scenic City Summit in Chattanooga, TN, and AmsterdamRB. Executive director André Arko went to RailsCamp to spread the word about Ruby Together. 
+
+In other community news, our Google Summer of Code students wrapped up their work on our various projects. Special thanks to our GSoC developers for all of their hard work these past few months!
+
+## bundler news
+
+And with the help of [@eanlain](github.com/eanlain), we shipped a new guide: [“How to use Bundler with Docker”](https://bundler.io/v1.16/guides/bundler_docker_guide.html).
+
+We also added a new contributor to the core team: [Stephanie Morillo](https://www.twitter.com). Stephanie has helped create new Bundler docs (like the troubleshooting RubyGems and Bundler TLS/SSL issues guide), and update existing ones. She’s also responsible for updating the Bundler contributor guidelines and trying to make Bundler docs more accessible to new contributors.
+
+This month, Bundler gained 58 new commits, contributed by 9 authors. There were 334 additions and 54 deletions across 30 files.
+
+## rubygems.org news
+
+In August, we blacklisted gem namespaces which had derivative spellings of Rails dependencies in response to CVE-2018-3779. We also made performance improvements to the “rubygems#show” and “version#index” pages.
+
+This month, RubyGems.org gained 7 commits from 2 authors, making 45 additions and 36 deletions across 9 files.
+
+## rubygems news
+
```

Comment:
> rubygems news

@indirect I know Luis worked on a bunch of stuff for RG last month; maybe we can highlight some of those bits?